Gaijinizer patch/Troops.txt

```diff
@@ -13368,10 +13365,10 @@ The rest is just sucked by me...#GT
 ダンスが始まる前から、おかしな事だらけだ…
 > CONTEXT: Troops/305/pages/1/334/Dialogue
 > CONTEXT: Troops/309/pages/1/332/Dialogue
-That's it... It's strange how you think about this...
-
-Before the dance began,
-there were lots of weird things...#GT
+That's right... This situation is strange no
+matter how you look at it. Before our dance
+started there were weird things all over the
+place...#Ron
 > END STRING
```

Comment:
We can probably squeeze this into three lines because we can use the extended Dialogue line length.